derive_builder/src/lib.rs

```diff
@@ -178,6 +178,34 @@
 //! - `#[builder(setter(skip = "true"))]`
 //! - `#[builder(setter(skip = "false"))]`
 //!
+//! ## Custom setters (skip autogenerated setters)
+//!
+//! Similarly to `setter(skip)`, you can say that you will provide your own setter methods.
+//! This simply suppresses the generation of the setter, leaveing the field in the builder,
+//! as `Option<T>`.
+//!
+//! ```rust
+//! # #[macro_use]
+//! # extern crate derive_builder;
+//! #
+//! #[derive(Builder)]
+//! struct SetterOptOut {
+//!     #[builder(setter(custom))]
+//!     custom_setter: u32,
+//! }
+//! impl SetterOptOutBuilder {
+//!     fn set_custom_setter(&mut self, value: u32) {
```

Comment:
Can you have this be named `custom_setter` to show that there is no generated setter this collides with?

Reply:
Done, thanks.